src/extension/features/accounts/calculate-irr/settings.js

```diff
@@ -0,0 +1,20 @@
+module.exports = [
+  {
+    name: 'CalculateIRR',
+    type: 'select',
+    default: 'disabled',
+    section: 'accounts',
+    title: 'Calculate Internal Rate of Return',
+    description:
+      'Calculate Internal Rate of Return for Tracking Accounts. Set Tracking Account contributions to this color to calculate IRR',
+    options: [
+      { name: 'disabled', value: null },
```

Comment:
once you rebase on `master`, you can remove this option and set `default: false` for this feature